runtime/resolvers/annotations.go

```diff
@@ -135,3 +135,7 @@ func (r *annotationsResolver) ResolveInteractive(ctx context.Context) (runtime.R
 func (r *annotationsResolver) ResolveExport(ctx context.Context, w io.Writer, opts *runtime.ResolverExportOptions) error {
 	return errors.New("not implemented")
 }
+
+func (r *annotationsResolver) InferRequiredSecurityRules() []*runtimev1.SecurityRule {
+	panic("not implemented")
```

Comment:
Maybe be safe and just return `nil`? The docstring says "attempts" so means it's not guaranteed.

Reply:
yeah did it safety reasons so that its missed not unintentionally, what do you think?